requirements/requirements.in

```diff
@@ -65,7 +65,7 @@ django-countries==4.6
 reportlab==3.0
 stripe==1.67.0
 django-compressor==2.1
-django-angular==0.7.16
+django-angular>=0.7.16
```

Comment:
Is this accurate? Can you import from both `djangular` and `djng` in 0.7.16?

Reply:
nope, you're right. Lemme figure this out and I'll update